src/core/client_channel/backup_poller.cc

```diff
@@ -63,16 +63,16 @@ static bool g_backup_polling_disabled;
 
 void grpc_client_channel_global_init_backup_polling() {
 #ifdef GRPC_PYTHON_BUILD
-  // Disable backup polling if EventEngine is used everywhere.
-  g_backup_polling_disabled =
-      !grpc_core::IsEventEnginePollerForPythonEnabled() &&
-      grpc_core::IsEventEngineClientEnabled() &&
-      grpc_core::IsEventEngineListenerEnabled() &&
-      grpc_core::IsEventEngineDnsEnabled();
+  bool python_poller_experiment_enabled =
+      grpc_core::IsEventEnginePollerForPythonEnabled();
 #else
-  // EventEngine polling not supported, keep using the backup poller.
-  g_backup_polling_disabled = false;
+  // Always on when not Python
+  bool python_poller_experiment_enabled = true;
 #endif
+  g_backup_polling_disabled = python_poller_experiment_enabled &&
+                              grpc_core::IsEventEngineClientEnabled() &&
+                              grpc_core::IsEventEngineListenerEnabled() &&
+                              grpc_core::IsEventEngineDnsEnabled();
```

Comment:
discussed IRL, may make more sense to be more explicit and copy-paste both conditions

Reply:
Done